pkg/controller/nodelifecycle/metrics.go

```diff
@@ -68,6 +68,9 @@ var (
 			Name:           evictionsTotalKey,
 			Help:           "Number of Node evictions that happened since current instance of NodeController started.",
 			StabilityLevel: metrics.STABLE,
+			// this is going to be replaced by taint_eviction_controller_pod_deletions_total, which has been
+			// implemented as part of KEP-3902 (Decouple TaintManager from NodeLifeCycleController)
+			DeprecatedVersion: "1.30.0",
```

Comment:
Why are we deprecating this metric? This one does not seem to be equivalent to the new ones (evicted/deleted pods). This one measures Node eviction occurrences (change in NodeReady condition) per zone.

Reply:
I think you are actually right. I misread the implementation and thought that it was counting the individual Pods being evicted but this is actually Nodes.

I have removed the DeprecatedVersion from here, thanks for spotting this! 🙏

Reply:
I see, thanks for the update